Cache/IO.lean

```diff
@@ -51,12 +59,23 @@ def CURLBIN :=
   -- change file name if we ever need a more recent version to trigger re-download
   IO.CACHEDIR / s!"curl-{CURLVERSION}"
 
+/-- leantar version at https://github.com/digama0/leangz -/
+def LEANTARVERSION :=
+  "0.1.2"
```

Comment:
This PR should wait until we have statically linked builds of leantar.

Reply:
```
$ ldd ~/.cache/mathlib/leantar-0.1.3
        statically linked
```